pyproject.toml

```diff
@@ -0,0 +1,12 @@
+[tool.poetry]
```

Comment:
Any specific reason to use poetry instead of PDM? 

The later is nearer to the contributor experience which we have already with PNPM.

Reply:
it' copy'n'past of docs repo. I don't know pdm yet, so feel free to suggest required changes.

Reply:
ok, will checkout pdm now